disaggregated_memory/cachehierarchies/dm_caches_sst.py

```diff
@@ -0,0 +1,218 @@
+# Copyright (c) 2023 The Regents of the University of California
```

Comment:
What are "dm" caches and why are they different from the normal cache hiearchies?